tendrl/commons/flows/create_cluster/__init__.py

```diff
@@ -52,9 +28,12 @@ def run(self):
             if all(status == "finished" for status in all_status):
                 all_ssh_jobs_done = True
 
-        # SSH setup jobs finished above, now install ceph
-        ceph_help.create_ceph(self.parameters)
+        # SSH setup jobs finished above, now install sds bits and create cluster
+        if self.parameters['sds_type'] == "ceph":
+            ceph_help.create_ceph(self.parameters)
+        else:
+            gluster_help.create_gluster(self.parameters)
 
     def load_definition(self):
```

Comment:
You cannot override load_definition(), either mark this flow with class variable "internal=True" or define this flow correctly in definitions (.yml) file